### DISCUSSION TITLE
Working with LoRa 32 V3

### DISCUSSION THREAD

**Discussion by jolitti** [micropython/micropython#10527] [ESP32]
After working for some time on an expressif ESP32, I formed a preference for micropython over C++ for low-intensity processing. Now I just received a couple of Heltec LoRa 32 V3 boards and I wanted to know if it's possible to use its LoRa capabilities with micropython.

Thanks for the attention

**Comment by jimmo** [comment]
@jolitti Yes, there are a range of different implementations of drivers for the various LoRa chips (e.g. SX126x, SX127x), including one in development in micropython-lib -- https://github.com/micropython/micropython-lib/pull/501.

I can't tell from the product page what chip your board uses.

If you're after LoRaWAN support rather than just direct LoRA then there are other third-party libraries available for this too.

**Reply by jolitti** [reply]
I've looked up the datasheet and the V3 uses a SX1262. Is the library usable at this point or should I wait before trying it?
